### File look OK, but a lot changed in style
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:clear-input.txt	2017-04-27 21:43:57.000000000 +0100
+++ saved-pages-clus/cl:functions:clear-input.txt	2017-04-27 21:43:57.000000000 +0100
@@ -2,1 +4,1 @@
 Syntax


-clear-input &optional input-stream => nil
+ clear-input &optional input-stream → nil

 Arguments and Values

@@ -14,50 +27,93 @@
 Examples


-;; The exact I/O behavior of this example might vary from implementation
-;; to implementation depending on the kind of interactive buffering that
-;; occurs.  (The call to SLEEP here is intended to help even out the
-;; differences in implementations which do not do line-at-a-time buffering.)
+The exact I/O behavior of this example might vary from implementation to implementation depending on the kind of interactive buffering that occurs. (The call to sleep here is intended to help even out the differences in implementations which do not do line-at-a-time buffering.)

 (defun read-sleepily (&optional (clear-p nil) (zzz 0))
-  (list (progn (print '>) (read))
-        ;; Note that input typed within the first ZZZ seconds
-        ;; will be discarded.
+  (list (progn (print '>)
+               (read)) ;; Note that input typed within the first
+                       ;; ZZZ seconds will be discarded.
         (progn (print '>)
-               (if zzz (sleep zzz))
-               (print '>>)
-               (if clear-p (clear-input))
+               (when zzz (sleep zzz))
+               (print '»)
+               (when clear-p (clear-input))
                (read))))

 (read-sleepily)
->>  > 10
->>  >
->>  >> 20
-=>  (10 20)
+▷
+> 10
+ 20 </o>
+→ (10 20)

 (read-sleepily t)
->>  > 10
->>  >
->>  >> 20
-=>  (10 20)
+▷
+> 10
+ 20 </o>
+→ (10 20)

 (read-sleepily t 10)
->>  > 10
->>  > 20  ; Some implementations won't echo typeahead here.
->>  >> 30
-=>  (10 30)

-<TT>
-Side Effects
+▷
+> 10
+ 20 ; Some implementations won't echo typeahead here.
+ 30 </o>
+→ (10 30)
+</blockquote>

+====Side Effects====
 The input-stream is modified.
-Affected By

+====Affected By====
 *standard-input*
-Exceptional Situations

+====Exceptional Situations====
 Should signal an error of type type-error if input-stream is not a stream designator.
-See Also

-clear-output
-Notes None.
\ No newline at end of file
+====See Also====
+ Function CLEAR-OUTPUT
+====Notes====
+None.
\ No newline at end of file